bin/migration/requirements.txt

```diff
@@ -1,14 +1,5 @@
-attrs==23.2.0
-importlib-metadata==7.1.0
-iniconfig==2.0.0
-packaging==24.0
-pluggy==1.4.0
-psycopg2==2.9.9
-py==1.11.0
-pyparsing==3.1.2
-pytest==8.1.1
-pytz==2024.1
-tomli==2.0.1
-typing_extensions==4.11.0
-zipp==3.18.1
-pandas==2.2.2
+azure_storage==0.37.0
+pandas==2.0.3
+psycopg2-binary==2.9.9
+pytz==2022.7
```

Comment:
@mazzopardi2 do we need both versions of pytz ?

Reply:
good spot thank you - removed the unnecessary one